api/app/routes/auth.py

```diff
@@ -0,0 +1,91 @@
+import logging
+
+from app import db
+from app.models.roles import Roles
+from app.models.user import User
+from flask import Blueprint
+from flask_apispec import marshal_with
+from flask_apispec import use_kwargs
+from flask_login import login_user
+from marshmallow import Schema
+from webargs import fields
+from werkzeug.security import check_password_hash
+from werkzeug.security import generate_password_hash
+
+
+blueprint = Blueprint("authorization", __name__)
+
+
+class LoginSchema(Schema):
+    password = fields.Str(required=True)
+    email_address = fields.Str(required=True)
+
+    class Meta:
+        strict = True
+
+
+class Login(object):
+    def __init__(self, password: str, email_address: str) -> None:
+        self.password = password
+        self.email_address = email_address
+
+
+class AuthResponseSchema(Schema):
+    message = fields.Str()
+
+
+class AuthResponse(object):
+    def __init__(self, message: str) -> None:
+        self.message = message
+
+
+@blueprint.route("/login", methods=["GET"])
+@use_kwargs(LoginSchema, apply=True)
+@marshal_with(AuthResponseSchema)
+def login(**kwargs):
+    user_login = Login(**kwargs)
+
+    user = User.query.filter_by(email_address=user_login.email_address).first()
+
+    logging.info("User attempting to login")
+
+    if not user or not check_password_hash(user.password, user_login.password):
```

Comment:
@bnorquist, let's be more specific with errors instead of combining them so we ourselves can debug better and the users are more informed

```suggestion
    if not user
      return AuthResponse('Username is not registered'), 404
      
    if not check_password_hash(user.password, user_login.password)
      return AuthResponse('Incorrect password'), 404 
```